README.md

```diff
@@ -342,6 +344,58 @@ py_image(
 To use a Python 3 runtime instead of the default of Python 2, use `py3_image`,
 instead of `py_image`.  The other semantics are identical.
 
+### nodejs_image
+
+**It is notable that unlike the other image rules, `nodejs_image` is not
+currently using the `gcr.io/distroless/nodejs` image for a handful of reasons.**
+This is a switch we plan to make, when we can manage it.  We are currently
+utilizing the `gcr.io/google-appengine/base` image as our base.
+
+To use `nodejs_image`, add the following to `WORKSPACE`:
+
+```python
+git_repository(
+    name = "build_bazel_rules_nodejs",
+    # Replace with a real commit SHA
+    commit = "{HEAD}",
+    remote = "https://github.com/bazelbuild/rules_nodejs.git",
+)
+
+load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories", "npm_install")
+
+# Download Node toolchain, etc.
+node_repositories(package_json = ["//:package.json"])
+
+# Install your declared Node.js dependencies
+npm_install(
+    name = "npm_deps",
```

Comment:
is this the canonical name users should always pick?

Reply:
Not necessarily, I'm just not very creative I guess :)